request/builder.js

```diff
@@ -66,8 +82,15 @@ class CertificateRegistrationRequestBuilder {
   /**
    * Builds the CertificateRegistrationRequest
    * @return {CertificateRegistrationRequest}
+   * @throws {Error}
    */
   async build() {
+    const validator = new Validator();
+    validator.validateInput(this.certHolderId, String);
+    validator.validateInput(this.certVersion, Number);
+    validator.validateInput(this.certPem, String);
```

Comment:
What will  happen if `certPem` is null ?

Reply:
Sorry, I thought we are still using `typeof` to test type.
In the way of `input.constructor === type`, we have to check if input is `null` or `undefined`.